code/game/objects/effects/decals/cleanable/aliens.dm

```diff
@@ -39,9 +39,12 @@
 	if(!step_to(src, get_step(src, direction), 0))
 		return
 	if(mapload)
-		for (var/i in 1 to range)
+		for (var/i = 1, i < range, i++)
```

Comment:
why?

Reply:
I made it copy human's streak because they are meant to be doing the same thing.
Human one was changed in https://github.com/tgstation/tgstation/pull/62567 and Xenos were never changed to match.

Reply:
I hate old me
I think `for(x in start to end)` is more readable in this sort of case, and is preferable unless the endpoint changes or something more complex is needed. could you change the outlier in human code?

Reply:
Done